thehoard/thehoard.github.io
# WIP - https://linear.app/the-hoard/issue/THE-23/user-can-choose-the-mini-base

CHANGES : 

-Design using bootstrap when relevant
-Switched to SCSS to allow nesting
-User can now choose a base for the mini from 4 choices : Grass, sand, cobblestone, lava

HOW TO TEST :

-run npm install
-run npm server
-go to your localhost
-you should see a stylized welcome screen with Logo, title, a parchment with a goblin and an introduction text. Next to the goblin you should see a portal.
-clicking on the portal, or dropping an image on it, should start the crop selection
-once the crop selection is complete, the editor option should appear
-Clicking on any editor option should update the preview instantly (including the Minibase style)
-Clicking on the "intégrer à l'armée" button should add the mini to the A4 Canvas, including the modified Minibase
-When you add more pages to the Army, the background image should repeat itself vertically
-Clicking the "télécharger l'armée" button should start a download of a pdf file with the current army on it

ISSUES :

-In progress : Design of bootstrap buttons
-In progress : the A4 canvas has a parchment color, it needs to be replaced with plain white when downloading the pdf file to avoid unecessary use of printer ink
-Design is not responsive
